commonFunctions.py

```diff
@@ -194,25 +195,44 @@ def dropout(self, value):
     else:
       raise TypeError("dropout must be a double")
 
+  @property
+  def l2(self):
+    """The 'l2' property"""
+    if self._verbose:
+      print "Getter of 'l2' called"
+    return self._l2
+  @l2.setter
+  def l2(self, value):
+    """Setter of the 'l2' property"""
+    if isinstance(value, (int, long, float)):
+      if value < 0 or value >= 1:
+        raise ValueError("Fraction must be between 0 and 1")
+      else:
+        self._l2 = float(value)
+    else:
+      raise TypeError("l2 must be a double")
+
   def buildModel(self, nIn, nOut, compileArgs):
     from keras.models import Sequential
     from keras.layers import Dense, Dropout, AlphaDropout
+    from keras.regularizers import l1,l2
+
     if self.type == "simple":
       model = Sequential()
-      model.add(Dense(self.neurons, input_dim=nIn, kernel_initializer='he_normal', activation=self.activation))
+      model.add(Dense(self.neurons, input_dim=nIn, kernel_initializer='he_normal', activation=self.activation, kernel_regularizer=l2(self.l2)))
```

Comment:
Should we maybe instead use the regularizer that allows for both l1 and l2 simultaneously? This means adding the l1 as an option too